client/src/components/RoommateAttributes/RoommateAttributes.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RoommateAttributes = ({ attribute, value }) => {
```

Comment:
This seens to just be one attribute, perhaps it should be called `RoommateAttribute`?